modules/ROOT/pages/munit-extensions-maven-plugin.adoc

```diff
@@ -0,0 +1,97 @@
+= Operating the MUnit Extensions Maven Plugin
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Learn how to run your tests using the MUnit Extensions Maven plugin.
+
+== Run MUnit Tests for a Module Project
+
+To run MUnit tests for your module:
+
+[source,console]
+----
+mvn clean verify
+----
+
+== Run a Specific MUnit Test Suite
+
+You can instruct the MUnit Extensions Maven plugin to run tests that only belong to a specific test suite by using the `munit.test` property.
+
+[source,console]
+----
+mvn clean verify -Dmunit.test=<regex-test-suite>
+----
+
+The `munit.test` property accepts regular expressions. The expression is applied to the name of the MUnit test suite file. The regular expression language is the Java implementation. This path is relative to `src/test/munit`.
+
+For example:
+
+[source,console]
+----
+mvn clean verify -Dmunit.test=.*my-test.*
+----
+
+You can leverage this feature by adding naming conventions to your MUnit test suites.
+
+== Run Specific MUnit Tests
+
+In the same way that you instruct MUnit to run one test suite, you can also instruct MUnit to run a specific test inside that test suite. To do so, use the `munit.test` property, with one addition:
+
+[source,console]
+----
+mvn clean verify -Dmunit.test=<regex-test-suite>#<regex-test-name>
+----
+
+The addition is the special character `#`. After `#`, you can type the test name. It also accepts regular expressions. The expression is applied to the attribute `name` of the MUnit test.
+
+For example:
+
+[source,console]
+----
+mvn clean verify -Dmunit.test=.*my-test.*#.*test-scenario-1.*
+----
+
+[TIP]
+--
+The tests inside the MUnit test suite that don't match the regular expression are flagged as *ignored*.
+--
+
+== Run Tests Using a Specific Tag
+
+You can choose to run only the tests that you grouped under one specific tag.
+
+[source,console]
+----
+mvn clean verify -Dmunit.tags=<munit-tag>
+----
+
+== Skip All Tests
+
+When building your module, you can prevent a test from running. MUnit leverages the same mechanism as Maven, so use the `skipTests` parameter to skip tests.
```

Comment:
```suggestion
When building your module, you can prevent a test from running. MUnit leverages the same mechanism as Maven, so use the `skipTests` parameter to skip tests:
```

Reply:
Do you need to say that MUnit leverages the same mechanism as Maven here? If not:
 "When building your module, you can prevent a test from running by using the `skipTests` parameter.